### ISSUE TITLE
Configuration saved from appended file should not affect main config file

### ISSUE THREAD

**Issue by natinusala** [libretro/RetroArch#1718]
When I edit some stuff in a config file that I append with `--append-config`, the main config file (in `~/.config/retroarch`) gets updated with the values in the appended file. If I want to revert back some entries to default, I have to remove them from the appended file AND from the main config file.

I think that everything overriden by an appended config file should not be updated in the main config file (or there should be an option to disable this) ; for me, the append config feature is for creating a custom config which overrides default for one session only, if you don't append it anymore it shouldn't impact the default behavior (by default I mean `retroarch.cfg`).

**Comment by natinusala** [comment]
And if you edit the config from the menu, it should be written back into where edited entries came from ; original or appended config file (or ask the user ?)

**Comment by fr500** [comment]
Same happened with overrides so I have made it so SAVING is disabled when overrides are active.
Can't you just use overrides for the time being? It's exactly the same as append config but automated (no need to specify --appendconfig on the commandline)

I explained the feature here: http://blog.andressm.org/new-retroarch-features-2/

**Comment by natinusala** [comment]
Okay, looks interesting, do you plan to support saving later or are you going to keep it this way ? Maybe we could add a file specific config property to specify if we want this particular file to be saved ?

**Comment by fr500** [comment]
Yeah eventually. Saving this delta configuration is quite complicated.

**Comment by zoltanvb** [comment]
Quite old ticket, but it still works the same way, so let me put a workaround here: disable config save from the append file, by adding this line there:
`config_save_on_exit = "false"`
It is definitely not as sophisticated as "saving each value to whichever config file it was coming from", but works for some use cases.
